stubs/pytest-lazyfixture/pytest_lazy_fixture.pyi

```diff
@@ -0,0 +1,14 @@
+from typing import Any, Iterable, overload
+
+class LazyFixture:
+    name: str
+    def __init__(self, name: str) -> None: ...
+    def __eq__(self, other: LazyFixture) -> bool: ...
```

Comment:
Mypy complains about this, because `__eq__` should handle any object, not just objects of the same type.

```suggestion
    def __eq__(self, other: object) -> bool: ...
```

The `__eq__` in pytest-lazy-fixture doesn't actually do this correctly. I created https://github.com/TvoroG/pytest-lazy-fixture/pull/53 to fix it.

Reply:
thanks